examples/search_examples/public/search_sessions/app.tsx

```diff
@@ -73,7 +73,7 @@ enum DemoStep {
   RestoreSessionViaManagement,
 }
 
-interface State extends QueryState {
+interface State extends Partial<QueryState> {
```

Comment:
Please see description on why existing consumers of `QueryState` have to use `Partial` now

Reply:
Not relevant anymore https://github.com/elastic/kibana/pull/132035#pullrequestreview-970639216